modules/imgproc/perf/perf_warp.cpp

```diff
@@ -231,7 +231,7 @@ PERF_TEST_P( TestRemap, remap,
         remap(source, destination, map_x, map_y, interpolationType, borderMode);
```

Comment:
8UC3 case is not covered by tests.
Perhaps we need to add test for 8UC3 (to keep perf tests set compact we can remove sz720p (it is highly correlates with 1080p)):
```
Values(CV_8UC1, CV_8UC3, CV_8UC4, CV_32FC1),
Values(szVGA, sz1080p),
```

Reply:
Updated the test condition